sdk/keyvault/Azure.Security.KeyVault.Secrets/samples/Sample4_ErrorResponse.md

```diff
@@ -0,0 +1,76 @@
+# Getting error details
+
+This sample demonstrates how get details from an error response from Azure Key Vault with soft delete enabled. You will attempt to create a secret but display a message to the user if the secret has been deleted but not purged.
+To get started, you'll need a URI to an Azure Key Vault. See the [README](https://github.com/Azure/azure-sdk-for-net/blob/main/sdk/keyvault/Azure.Security.KeyVault.Secrets/README.md) for links and instructions.
+
+> Note: Details of an error apart from `code` or `message` are not part of the service contract and should not be used for mission-critical decisions. The error code and message are conveniently defined as properties on `RequestFailedException`.
+
+## Creating a SecretClient
+
+To create a new `SecretClient` to attempt to set a secret, you need the endpoint to an Azure Key Vault and credentials.
+You can use the [DefaultAzureCredential](https://github.com/Azure/azure-sdk-for-net/blob/main/sdk/identity/Azure.Identity/README.md) to try a number of common authentication methods optimized for both running as a service and development.
+
+In the sample below, you can set `keyVaultUrl` based on an environment variable, configuration setting, or any way that works for your application.
+
+```C# Snippet:SecretsSample1SecretClient
+var client = new SecretClient(new Uri(keyVaultUrl), new DefaultAzureCredential());
+```
+
+## Parsing error details
+
+Azure Key Vault will return an error response if you attempt to create a secret, key, or certificate that is in a deleted state.
+If the service returns an HTTP 409 (Conflict) error, you can use `RequestFailedException.GetRawResponse()` to get the raw HTTP response and parse the content for more details.
```

Comment:
nit: 
```suggestion
If the service returns an HTTP 409 (Conflict) error, `RequestFailedException.GetRawResponse()` can be called to get the raw HTTP response and parse the content for more details.
```

Reply:
Microsoft Learn guidance is to use the informal first-person active voice.

Reply:
Ah, good to know!